srcpkgs/python3-pgzero/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-pgzero'
+pkgname=python3-pgzero
+version=1.2
+revision=1
+wrksrc="pgzero-${version}"
+build_style=python3-module
+pycompile_module="pgzero pgzrun.py"
+archs=noarch
```

Comment:
should be above wrksrc=

Please use `xlint` to find these common errors.

Reply:
Ok, fixed.  I had run `xlint` on the previous commits, just not the latest one... oops!